controller/workitemtype.go

```diff
@@ -16,25 +16,39 @@ const (
 // WorkitemtypeController implements the workitemtype resource.
```

Comment:
Skratch that.. It's the target Location from the POST that should have the cache info..  hmm..